rasa_core/tracker_store.py

```diff
@@ -136,3 +139,58 @@ def retrieve(self, sender_id):
             return self.deserialise_tracker(sender_id, stored)
         else:
             return None
+
+
+class MongoTrackerStore(TrackerStore):
+    def __init__(self,
+                 domain,
+                 host="mongodb://localhost:27017",
+                 db="rasa",
+                 username=None,
+                 password=None,
+                 collection="conversations",
+                 event_broker=None):
+        from pymongo.database import Database
+        from pymongo import MongoClient
+
+        self.client = MongoClient(host,
+                                  username=username,
+                                  password=password,
+                                  # delay connect until process forking is done
+                                  connect=False)
+
+        self.db = Database(self.client, db)
+        self.collection = collection
+        super(MongoTrackerStore, self).__init__(domain, event_broker)
+
+        self._ensure_indices()
+
+    def _conversations(self):
+        return self.db[self.collection]
+
+    def _ensure_indices(self):
```

Comment:
`_ensure_index()` as it's only one?

Reply:
it's called like this everywhere else, lets keep the plural - maybe we want to add another index in the future :D .